NOpenType/IGlyphRasterizer.cs

```diff
@@ -4,6 +4,8 @@ namespace NRasterizer
 {
     public interface IGlyphRasterizer
     {
+        int Resolution { get; }
+
```

Comment:
This also feels slightly malplaced. It would be awesome if the `IGlyphRasterizer` was unit-agnostic. But then how _would_ one pass the target resolution to the scaling code... Hmm, it makes a lot of sense. Let's do it like this. :+1: